plugins/woocommerce/src/Admin/Features/MultichannelMarketing/Init.php

```diff
@@ -1,87 +0,0 @@
-<?php
-/**
- * Multichannel Marketing Experience
- *
- * @package Woocommerce Admin
- */
-
-namespace Automattic\WooCommerce\Admin\Features\MultichannelMarketing;
-
-use Automattic\WooCommerce\Internal\Admin\Survey;
-use Automattic\WooCommerce\Admin\Features\Features;
-use Automattic\WooCommerce\Internal\Admin\WCAdminAssets;
-
-/**
- * Contains logic for Multichannel Marketing.
- */
-class Init {
-	/**
-	 * Option name used to toggle this feature.
-	 */
-	const TOGGLE_OPTION_NAME = 'woocommerce_multichannel_marketing_enabled';
```

Comment:
Could this tracking be removed as well?

https://github.com/woocommerce/woocommerce/blob/046b0195ab6ba23733dc0e28ab9be047d496ea61/plugins/woocommerce/includes/class-wc-tracker.php#L699

Reply:
This has been done in PR https://github.com/woocommerce/woocommerce/pull/37438.